ResourceStatusSystem/Command/FreeDiskSpaceCommand.py

```diff
@@ -20,9 +20,9 @@ class FreeDiskSpaceCommand( Command ):
   Uses diskSpace method to get the free space
   '''
 
-  def __init__( self, args = None ):
+  def __init__( self, args = None, clients = None ):
 
-    super( FreeDiskSpaceCommand, self ).__init__( args )
+    super( FreeDiskSpaceCommand, self ).__init__( args, clients )
```

Comment:
You mean like this? `super( FreeDiskSpaceCommand, self ).__init__( args, clients = clients )`